R/utils.r

```diff
@@ -102,6 +102,19 @@ print_yaml <- function(x) {
   structure(x, class = "print_yaml")
 }
 
+pkgdown_field <- function(pkg, ...) {
+  field <- paste0(list(...), collapse = ".")
+  config <- src_path(
+    fs::path_rel(
+      pkgdown_config_path(pkg$src_path),
+      pkg$src_path
+    )
+  )
+  paste0(
+    crayon::bold(field), " in ", config
```

Comment:
this assumes use of config only, not override. should the string actually be "blabla field in <config-path> (or `override` parameter)"? :thinking: